source/iface/common-byval.rst

```diff
@@ -9,13 +9,14 @@
 ====================
 
 Types: :ref:`id`, :ref:`range`, :ref:`item`, :ref:`nd_item`,
-:ref:`h_item`, :ref:`group`, and :ref:`nd_range`.
+:ref:`h_item`, :ref:`group`, `sub_group`, and :ref:`nd_range`.
```

Comment:
Just curious, why `:ref:` everywhere but not on this one?

Reply:
sub_group is not documented yet. I have been documenting sycl 2020 features when I use them so I can add an example.